seed/migrations/0198_auto_20230515_1405.py

```diff
@@ -0,0 +1,527 @@
+# Generated by Django 3.2.19 on 2023-05-15 17:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seed', '0197_ubidmodel'),
+    ]
+
+    operations = [
+
+        migrations.RunSQL(
```

Comment:
I ran into a few errors attempting to import, read, and execute an entire SQL file. I'd like to refactor this to leverage imports, but I didn't want to let it become a blocker.